### ISSUE TITLE
Doc mentions MAP but package doesn't allow it

### ISSUE THREAD

**Issue by gdalle** [JuliaStats/Distributions.jl#1559]
[This page of the doc](https://juliastats.org/Distributions.jl/latest/fit/#Maximum-a-Posteriori-Estimation) mentions Maximum-A-Posteriori Estimation, but I think it may be outdated since Distributions.jl doesn't seem to allow it. Maybe it could be removed?

**Comment by devmotion** [comment]
AFAICT `fit_map` was moved to https://github.com/JuliaStats/ConjugatePriors.jl (which would also explain the broken link).

**Comment by devmotion** [comment]
Hmm yes it seems it has to be updated a bit (which seems totally possible though, I left some comments in @mschauer's PR). I think ideally we would point to ConjugatePriors in that section but maybe currently users should be warned that it was neglected a bit and needs more love.

**Comment by gdalle** [comment]
Yeah I'm trying to clean up that PR and get it production ready in the next few weeks

**Comment by gdalle** [comment]
For future reference, here is the PR in question: https://github.com/JuliaStats/ConjugatePriors.jl/pull/43

**Comment by Dropnspl** [comment]
https://cash.app/$10gost88
